### PR TITLE
Improved comments and updated modules

### PR DESCRIPTION
**Pull request by noborus**

### CODE REVIEW

File: oviewer/control.go [hunk 1]
```diff
@@ -238,6 +238,8 @@ func (m *Document) requestLoad(chunkNum int) {
 }
 
 // requestLoadSync sends instructions to load chunks into memory.
+// requestLoadSync is a synchronous version of requestLoad.
```
Comment (ccoVeille) [inline oviewer/control.go:241]:
```suggestion
// requestLoadSync is a synchronous version of [requestLoad].
```

File: oviewer/document.go [hunk 2]
```diff
@@ -211,7 +219,8 @@ type LineC struct {
 	eolStyle tcell.Style
 }
 
-// NewDocument returns Document.
+// NewDocument creates and initializes a new Document with default settings.
```
Comment (ccoVeille) [inline oviewer/document.go:222]:
```suggestion
// NewDocument creates and initializes a new [Document] with default settings.
```

File: oviewer/document.go [hunk 3]
```diff
@@ -582,3 +598,20 @@ func (m *Document) setColumnWidths() {
...
+	}
+	return string(s)
+}
+
+// LineString returns one line from buffer.
+// Deprecated: Use LineStr instead.
+func (m *Document) LineString(n int) string {
+	str, _ := m.LineStr(n)
+	return str
+}
```
Comment (ccoVeille) [inline oviewer/document.go:617]:
```suggestion
// GetLine returns one line from buffer.
// Deprecated: Use [Document.LineString] instead.
func (m *Document) GetLine(n int) string {
	s, err := m.Line(n)
	if err != nil {
		return ""
	}
	return string(s)
}

// LineString returns one line from buffer.
// Deprecated: Use [Document.LineStr] instead.
func (m *Document) LineString(n int) string {
	str, _ := m.LineStr(n)
	return str
}
```

**Review by ccoVeille — COMMENTED** [review]
You could use godoc link (available since Go 1.19 

Here are some

**Comment by noborus** [comment]
> You could use godoc link (available since Go 1.19
> 
> Here are some

nice! thank you.

**Comment by ccoVeille** [comment]
Use `go doc -open` (something like that) to check locally

I'm unsure [readLink] worked [Document.readalink] might the right syntax

**Comment by noborus** [comment]
> Use `go doc -open` (something like that) to check locally
> 
> I'm unsure [requestLoad] worked [Document.requestLoad] might the right syntax

thank you. I confirmed it.

**Pull request merged**